runtime/autoload/remote/host.vim

```diff
@@ -120,9 +118,25 @@ function! remote#host#RegisterPlugin(host, path, specs) abort
 endfunction
 
 
+function! s:GetManifest() abort
+  if !exists('s:remote_plugins_manifest')
```

Comment:
This should check `filereadable()`, and then there is no need for `s:remote_plugins_manifest` to exist anymore.